Package.swift

```diff
@@ -14,24 +14,34 @@ let package = Package(
         .library(name: "Amplitude", targets: ["Amplitude"]),
     ],
     targets: [
-        .target(
-            name: "Amplitude",
-            path: "Sources/Amplitude",
-            exclude: ["AppledocSettings.plist"],
+          .target(
+            name: "EventExplorer",
+            path: "Sources/EventExplorer",
             resources: [
                 .process("Resources/AMPBubbleView.xib"),
                 .process("Resources/AMPInfoViewController.xib"),
-                .process("Resources/cancel.png"),
-                .process("Resources/cancel@2x.png"),
-                .process("Resources/cancel@3x.png"),
-                .process("Resources/ComodoRsaDomainValidationCA.der"),
-                .process("Resources/logo-banner.png"),
-                .process("Resources/logo-banner@2x.png"),
-                .process("Resources/logo-banner@3x.png"),
-                .process("Resources/logo-button.png"),
-                .process("Resources/logo-button@2x.png"),
-                .process("Resources/logo-button@3x.png")
+                .process("Resources/Images/cancel.png"),
+                .process("Resources/Images/cancel@2x.png"),
+                .process("Resources/Images/cancel@3x.png"),
+                .process("Resources/Images/logo-banner.png"),
+                .process("Resources/Images/logo-banner@2x.png"),
+                .process("Resources/Images/logo-banner@3x.png"),
+                .process("Resources/Images/logo-button.png"),
+                .process("Resources/Images/logo-button@2x.png"),
+                .process("Resources/Images/logo-button@3x.png")
             ],
-            publicHeadersPath: ".")
+            publicHeadersPath: ".",
+            cSettings: [
+                .headerSearchPath("../Amplitude")
```

Comment:
What's this for?

Reply:
[`EventExplorer` uses files outside its group](https://github.com/Amplitude/Amplitude-iOS/blob/macOS-tvOS-disable-event-explorer/Sources/EventExplorer/AMPInfoViewController.m#L26). If we don't configure `headerSearchPath`, then `EventExplorer` won't be able to find them


Build failure example with no `headerSearchPath`:

![image](https://user-images.githubusercontent.com/15751908/99201895-0b03aa80-2762-11eb-90c5-bdf7f4468d8a.png)